test/simulator/misc/statuses.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const assert = require('./../../assert');
-const common = require('./../../common');
+const assert = require('../../assert');
+const common = require('../../common');
```

Comment:
@Slayer95 @Zarel yeah so the reason I changed it was because this is the only place I have seen require used with that sort of prefix.

`../` is already clear that it is a relative require.

That said, if this is something that is done consistently across the entire codebase, I'm more than happy to change it back to `./../`, just in my eyes it feels fairly redundant.